content/doc/book/installing/_docker.adoc

```diff
@@ -7,6 +7,12 @@ If you update content on this page, please ensure the changes are reflected in
 the sibling file _docker-for-tutorials.adoc (used in _run-jenkins-in-docker.adoc).
 ////
 
+== Java Requirement
+
+Jenkins requires Java 17 or Java 21 (64-bit).  
+Install Java before proceeding.  
+Refer to the [_Installation Requirements_](./_installation_requirements.adoc) page for details.
+
```

Comment:
Java is already included in the container definition.  Declaring that Java 17 or Java 21 should be installed first does not make sense in this page because the container image already includes the correct Java version
```suggestion
```